MAUI/Kanban-Board/Placeholder.md

```diff
@@ -37,43 +37,43 @@ The following code example describes the above behavior.
 
```

Comment:
@EzhilarasanElangovan31  as we are not providing place holder style support in vol 3, remove this md file from toc file

Reply:
removeed this md file from toc file